p2p-specs/p2p-interface.md

```diff
@@ -251,10 +247,6 @@ however, certain encodings like SSZ do, for added security.
 
 A `response` is formed by zero or more `response_chunk`s.
 
-Responses that consist of a single SSZ-list send each list item as a `response_chunk`.
```

Comment:
This isn't true for the `Status` message. We want that to be a single response chunk.